src/search-manager/data/api.ts

```diff
@@ -246,7 +246,6 @@ export async function fetchSearchResults({
     highlightPreTag: HIGHLIGHT_PRE_TAG,
     highlightPostTag: HIGHLIGHT_POST_TAG,
     attributesToCrop: ['content'],
-    cropLength: 20,
```

Comment:
I changed to the default (10) to reduce the text in the description. The design asked to limit to 3 lines, but we can only control word count here. We can fine-tune this in the AC test review.